docs/CHANGELOG.rst

```diff
@@ -57,6 +57,7 @@ Fixed
   duplicated amplicon names are included. Validation will now pass also
   for primer sequences in lowercase.
 - Fix allele frequency (AF) calculation in ``snpeff`` process
+- Fix bug in script for calculating FPKM
```

Comment:
Provide more info about the bug.

Reply:
The info should be such that when the CS, product or marketing read about the fix in release notes, they know what it is about. I know you are part of the CS team, but the description should be such that the message does not have to be explained verbally.

Reply:
I'm on it.